migrator/migrations/policymigrationhelper/postgres_policy_migrator.go

```diff
@@ -203,3 +203,68 @@ func MigratePoliciesWithStoreV2(policiesToMigrate map[string]PolicyChanges,
 	}
 	return nil
 }
+
+// MigratePolicyCategoriesWithStoreV2 migrates policies and adds policy categories to policy proto with the given diffs.
```

Comment:
Do we not need to add the edge since we are technically re-adding the category to the policy?